lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -7,33 +7,33 @@
 from lumicks.pylake.population.dwelltime import DwelltimeModel
 
 
-def export_kymolinegroup_to_csv(filename, kymoline_group, delimiter, sampling_width):
+def export_kymotrackgroup_to_csv(filename, kymotrack_group, delimiter, sampling_width):
```

Comment:
You capitalize the T in the class. Shouldn't this then my `kymo_track_group` for consistency (I do realize we may have been inconsistent before :D)?

I also kind of feel like this function name needs more underscores. `export_kymo_track_group_to_csv`? Or `export_kymotrack_group_to_csv`, but then the class should probably not capitalize track. What do you prefer?

Reply:
In general I would agree with that convention for renaming from CamelCase to snake_case however, in this particular instance I think it's worthwhile to break convention because 5 `_`s in a function name really seems like overkill; I think the name actually is way too long even as it is.

Had a conversation with Dean and he pointed out that the more pythonic thing to do would be make `KymoTrackGroup` public and expose the import/export functionality as methods, in which case you'd have `KymoTrackGroup.export_to_csv()`. I like this idea so maybe we can have a chat about it, because there are some additional concerns to keep in mind; at any rate that's for a separate PR

Reply:
Yeah, that needs some more other preparation first, but sure. I do have some ideas how to handle some of this as well because indeed, there's a fair amount of behaviour to decide on first. Maybe put a ticket on the backlog for that so we can refine it. For now this is fine.